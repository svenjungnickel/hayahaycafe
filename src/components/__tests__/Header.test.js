import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

jest.mock('../Image');

describe('Header', () => {
    it('renders without subtitle', () => {
        const headerImage = 'headerImage';
        const title = 'title';

        const component = <Header headerImage={headerImage} title={title} />;
        const tree = renderer.create(component).toJSON();

        expect(tree).toMatchSnapshot();
    });

    it('renders with subtitle', () => {
        const headerImage = 'headerImage';
        const title = 'title';
        const subtitle = 'subtitle';

        const component = <Header headerImage={headerImage} title={title} subtitle={subtitle} />;
        const tree = renderer.create(component).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
