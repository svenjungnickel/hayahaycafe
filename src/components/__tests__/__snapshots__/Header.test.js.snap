// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Header renders with subtitle 1`] = `
<header
  className="header"
>
  <img
    alt="alt"
    src="src"
  />
  <div
    className="header__content"
  >
    <div
      className="container"
    >
      <div
        className="header__contentInner"
      >
        <h1
          className="display-1"
        >
          title
        </h1>
        <p
          className="lead"
        >
          subtitle
        </p>
      </div>
    </div>
  </div>
</header>
`;

exports[`Header renders without subtitle 1`] = `
<header
  className="header"
>
  <img
    alt="alt"
    src="src"
  />
  <div
    className="header__content"
  >
    <div
      className="container"
    >
      <div
        className="header__contentInner"
      >
        <h1
          className="display-1"
        >
          title
        </h1>
      </div>
    </div>
  </div>
</header>
`;
